<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyhoteeMainWindow</class>
 <widget class="QMainWindow" name="KeyhoteeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyhotee</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="TreeWidgetCustom" name="side_bar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>220</red>
             <green>225</green>
             <blue>232</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>237</red>
             <green>237</green>
             <blue>237</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="indentation">
        <number>13</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>MAILBOXES</string>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>91</red>
           <green>91</green>
           <blue>91</blue>
          </color>
         </brush>
        </property>
        <item>
         <property name="text">
          <string>Inbox</string>
         </property>
         <property name="icon">
          <iconset resource="Keyhotee.qrc">
           <normaloff>:/images/inbox.png</normaloff>:/images/inbox.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drafts</string>
         </property>
         <property name="icon">
          <iconset resource="Keyhotee.qrc">
           <normaloff>:/images/drafts.png</normaloff>:/images/drafts.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Outbox</string>
         </property>
         <property name="icon">
          <iconset resource="Keyhotee.qrc">
           <normaloff>:/images/outbox.png</normaloff>:/images/outbox.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sent</string>
         </property>
         <property name="icon">
          <iconset resource="Keyhotee.qrc">
           <normaloff>:/images/128x128/sent_items.png</normaloff>:/images/128x128/sent_items.png</iconset>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WALLETS</string>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>91</red>
           <green>91</green>
           <blue>91</blue>
          </color>
         </brush>
        </property>
        <item>
         <property name="text">
          <string>Bitcoin</string>
         </property>
         <property name="icon">
          <iconset resource="Keyhotee.qrc">
           <normaloff>:/images/bitcoin.png</normaloff>:/images/bitcoin.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BitShares</string>
         </property>
         <property name="icon">
          <iconset resource="Keyhotee.qrc">
           <normaloff>:/images/bitshares.png</normaloff>:/images/bitshares.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Litecoin</string>
         </property>
         <property name="icon">
          <iconset resource="Keyhotee.qrc">
           <normaloff>:/images/litecoin128.png</normaloff>:/images/litecoin128.png</iconset>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CONTACTS</string>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>127</red>
           <green>127</green>
           <blue>127</blue>
          </color>
         </brush>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>91</red>
           <green>91</green>
           <blue>91</blue>
          </color>
         </brush>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>REQUESTS</string>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>127</red>
           <green>127</green>
           <blue>127</blue>
          </color>
         </brush>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>91</red>
           <green>91</green>
           <blue>91</blue>
          </color>
         </brush>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
      <widget class="QStackedWidget" name="widget_stack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>40</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>6</number>
       </property>
       <widget class="Mailbox" name="inbox_page">
        <layout class="QGridLayout" name="inboxlayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="Mailbox" name="draft_box_page"/>
       <widget class="Mailbox" name="out_box_page"/>
       <widget class="Mailbox" name="sent_box_page"/>
       <widget class="ContactsTable" name="contacts_page">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="ContactView" name="new_contact"/>
       <widget class="Wallets" name="wallets"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuMail">
    <property name="title">
     <string>Mail</string>
    </property>
    <addaction name="actionNew_Message"/>
    <addaction name="actionReply"/>
    <addaction name="actionReply_all"/>
    <addaction name="actionForward"/>
    <addaction name="separator"/>
    <addaction name="actionSave_attachement"/>
   </widget>
   <widget class="QMenu" name="menuContact">
    <property name="title">
     <string>Contact</string>
    </property>
    <addaction name="actionNew_Contact"/>
    <addaction name="actionSet_Icon"/>
    <addaction name="separator"/>
    <addaction name="actionRequest_authorization"/>
    <addaction name="actionShare_contact"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Contacts"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDiagnostic"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuIdentity">
    <property name="title">
     <string>Identity</string>
    </property>
    <addaction name="actionNew_identity"/>
    <addaction name="separator"/>
    <addaction name="actionEnable_Mining"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuIdentity"/>
   <addaction name="menuMail"/>
   <addaction name="menuContact"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Message"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Contact"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionShow_details"/>
  </widget>
  <action name="actionMail">
   <property name="text">
    <string>Mail</string>
   </property>
  </action>
  <action name="actionNew_Contact">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/add_user.png</normaloff>:/images/add_user.png</iconset>
   </property>
   <property name="text">
    <string>New Contact</string>
   </property>
  </action>
  <action name="actionNew_Message">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/email_compose.png</normaloff>:/images/email_compose.png</iconset>
   </property>
   <property name="text">
    <string>Compose</string>
   </property>
  </action>
  <action name="actionShow_Contacts">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/address_book.png</normaloff>:/images/address_book.png</iconset>
   </property>
   <property name="text">
    <string>Show Contacts</string>
   </property>
  </action>
  <action name="actionEnable_Mining">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enable Mining</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/win/editcopy.png</normaloff>:/images/win/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/win/editcut.png</normaloff>:/images/win/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/win/editpaste.png</normaloff>:/images/win/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/delete_icon.png</normaloff>:/images/delete_icon.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="statusTip">
    <string>Allows to delete currently selected item(s)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/mail_forward.png</normaloff>:/images/mail_forward.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionReply">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/mail_reply.png</normaloff>:/images/mail_reply.png</iconset>
   </property>
   <property name="text">
    <string>Reply</string>
   </property>
  </action>
  <action name="actionReply_all">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/mail_reply_all.png</normaloff>:/images/mail_reply_all.png</iconset>
   </property>
   <property name="text">
    <string>Reply all</string>
   </property>
  </action>
  <action name="actionSave_attachement">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/win/filesave.png</normaloff>:/images/win/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save attachment</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDiagnostic">
   <property name="text">
    <string>Diagnostic </string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_identity">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/add_contact.png</normaloff>:/images/add_contact.png</iconset>
   </property>
   <property name="text">
    <string>New identity</string>
   </property>
  </action>
  <action name="actionSet_Icon">
   <property name="text">
    <string>Set Icon</string>
   </property>
  </action>
  <action name="actionShow_details">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/showDetails.png</normaloff>:/images/showDetails.png</iconset>
   </property>
   <property name="text">
    <string>Show details</string>
   </property>
  </action>
  <action name="actionRequest_authorization">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/128x128/contact_info_request_authorisation.png</normaloff>:/images/128x128/contact_info_request_authorisation.png</iconset>
   </property>
   <property name="text">
    <string>Request authorization...</string>
   </property>
  </action>
  <action name="actionShare_contact">
   <property name="icon">
    <iconset resource="Keyhotee.qrc">
     <normaloff>:/images/128x128/contact_share.png</normaloff>:/images/128x128/contact_share.png</iconset>
   </property>
   <property name="text">
    <string>Share contact...</string>
   </property>
  </action>
  <action name="actionPolish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polish</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionPortuguese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Portuguese</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionSpanish_Chile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spanish (Chile)</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactView</class>
   <extends>QWidget</extends>
   <header>AddressBook/ContactView.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ContactsTable</class>
   <extends>QWidget</extends>
   <header>AddressBook/ContactsTable.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Mailbox</class>
   <extends>QWidget</extends>
   <header>Mail/Mailbox.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Wallets</class>
   <extends>QWidget</extends>
   <header>Wallets/wallets.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeWidgetCustom</class>
   <extends>QTreeWidget</extends>
   <header>TreeWidgetCustom.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Keyhotee.qrc"/>
 </resources>
 <connections/>
</ui>
